cpp/src/community/detail/common_methods.cuh

```diff
@@ -203,16 +209,8 @@ graph_contraction(raft::handle_t const& handle,
                         numbering_indices.size(),
                         new_graph_view.local_vertex_partition_range_first());
 
```

Comment:
`new_graph_view` is used in nowhere.

We're filling `numbering_indices` but it seems like this is not used anywhere as well? Better delete this?